### ISSUE TITLE
[0.7.0] -> [0.8.0] Dictionary regression

### ISSUE THREAD

**Issue by bbenezech** [pelotom/runtypes#7]
Considering

```typescript
const Errors = RT.Dictionary(RT.Array(RT.String))
type ErrorsType = RT.Static<typeof Errors>
```

`0.7.0` **correct**
```typescript
type ErrorsType = {
    [_: string]: string[];
}
```

`0.8.0` **incorrect**
```typescript
type ErrorsType = {
    [_: string]: Arr<String>;
}
```

Do you know intuitively what is causing this or would you like me to investigate further on?

Not urgent, I can stick on 0.7.0 for a while, it works perfectly for me.

Thanks again.

**Comment by gcanti** [comment]
Here https://github.com/pelotom/runtypes/blob/master/src/types/dictionary.ts#L5 `V` should be `Static<V>` I guess

```diff
-export interface StringDictionary<V extends Rt> extends Runtype<{ [_: string]: V }> {
+export interface StringDictionary<V extends Rt> extends Runtype<{ [_: string]: Static<V> }> {
```

Also https://github.com/pelotom/runtypes/blob/master/src/types/dictionary.ts#L11

**Comment by pelotom** [comment]
Thanks for the fix, @gcanti. Should be fixed in v0.9.1.

**Comment by bbenezech** [comment]
Thans @pelotom @gcanti 

Tested & works, awesome. 👍
